test-network-k8s/README.md

```diff
@@ -0,0 +1,75 @@
+# Kubernetes Test Network 
+
+This project re-establishes the Hyperledger [test-network](../test-network) as a _production deployment_ on Kubernetes.
+
+_Fabric, Ahoy!_
+
+
+### Objectives:
+
+- Provide **simple**, "one-click" actions for deploying production-grade Fabric networks on Kubernetes.
+- Provide a reference Kubernetes sample, illustrating community recommendations for CAs, MSP, and operational activities.  
+- Align dev, test, CI/CD, and production pipelines on a _common_, cloud-native runtime (Kubernetes).
+- Support a transition from Internal, [Docker daemon](link) to [External Chaincode](link) builders.
+- Support a transition to [Chaincode as a Service](link) deployments
+- Provide a K8s reference and supplement to Fabric [CA Operations](link) and [CA Deployment](link) guides.
+- Kube-agnostic : AWS, Azure, gcp, IKS, KIND, OCP, ... 
+
+
+## Prerequisites 
+
+- [Docker](https://www.docker.com)
+- [kubectl](https://kubernetes.io/docs/tasks/tools/)
+- [KIND](https://kind.sigs.k8s.io/docs/user/quick-start/#installation)
+
+
+## Quickstart
```

Comment:
step 0 clone the repo..... obvious I know.. but somepeople.

Reply:
Well.. on the bit.ly link we do have one, though the general consensus is to move away from that to an actual real URL...